cpp/IceBox/greeter/Chatbot.cpp

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) ZeroC, Inc.
```

Comment:
Greeter.ice, Chatbot.h and Chatbot.cpp are exact copies of the files in Ice/greeter.